### PR TITLE
Add VRV and Repl.it

### PR DESCRIPTION
**Pull request by shawnshyguy**
Added VRV (entertainment) and Repl.it (developer), neither of which have 2FA yet.

### CODE REVIEW

File: _data/entertainment.yml [hunk 1]
```diff
@@ -204,6 +204,12 @@ websites:
     twitter: Vimeo
     facebook: Vimeo
     img: vimeo.png
+ 
+  - name: VRV
+    url: https://www.vrv.co
+    img: vrv.png
+    twitter: WatchVRV
+    facebook: WatchVRV
```
Comment (Carlgo11) [inline _data/entertainment.yml:213]:
```suggestion
    regions:
      - us

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
1. For future reference, please only create one PR per site addition.
2. The images you use are unnecessarily large. Here are some smaller images:
vrv
replit

File: _data/entertainment.yml [hunk 2]
```diff
@@ -204,6 +204,12 @@ websites:
...
```
Comment (Carlgo11) [inline _data/entertainment.yml:212]:
```suggestion
    facebook: WatchVRV
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Also, `replit.png` should be in `img/developer`, not `img`.

**Comment by Carlgo11** [comment]
@shawnshyguy please fix the requested changes within a couple of days or the PR will be closed due to inactivity.

**Pull request closed without merging**